framework/include/postprocessors/SideIntegralFunctorPostprocessor.h

```diff
@@ -53,4 +49,11 @@ class SideIntegralFunctorPostprocessorTempl : public SideIntegralPostprocessor
 
   /// Whether to skip integrating where the functors are not both defined
   const bool _partial_integral;
+
+private:
+  template <typename T>
+  Real computeLocalContribution(const T & functor_arg) const;
```

Comment:
Very nice that you made this private. This ensures that you don't have to try and anticipate possible instantiations a derived class might make ... which would lead to undefined references